src/main/kotlin/com/terraformation/backend/accelerator/ApplicationCountryUpdater.kt

```diff
@@ -22,7 +24,7 @@ class ApplicationCountryUpdater(
     systemUser.run {
       val variable = variableStore.fetchOneVariable(event.variableId)
 
-      if (variable.stableId == STABLE_ID_COUNTRY) {
+      if (StableId(variable.stableId) == StableIds.country) {
```

Comment:
It will probably make sense to change `Variable` so that the `stableId` field is of type `StableId` rather than type `String`, since we're introducing a type wrapper for stable IDs, so we aren't having to wrap and unwrap the values regularly. But that can be its own separate PR.